src/configure/utilities/templateConverter.ts

```diff
@@ -0,0 +1,106 @@
+const templateParametersRegex = /(inputs|variables|assets|secrets)\.\w+/g;
+const templateEnclosedParametersRegex = /{{{?\s*(inputs|variables|assets|secrets)\.\w+\s*}?}}/g;
+const mustacheHelperRegex = /{?{{\s*#(\w+)(.*?)({?{{\s*\/\1)}}}?/;
```

Comment:
any reason why `{?` is applied to first `{` char and not third one like done one line above

Reply:
`{?` means optional, it is there to match parameters even if they are 2 or 3 braces.